torch/distributed/distributed_c10d.py

```diff
@@ -4654,30 +4654,43 @@ def barrier(
         group (ProcessGroup, optional): The process group to work on. If None,
             the default process group will be used.
         async_op (bool, optional): Whether this op should be an async op
-        device_ids ([int], optional): List of device/GPU ids.
+        device_ids ([int], optional): List of device/GPU ids. Only one id is expected.
```

Comment:
Can we change this to

```
Only the first ID is used.
```

Reply:
I do mean only one is expected, because now we are expecting one device per thread. Some of the API signatures came from the old days.